docs/guides/DEVELOPMENT.md

```
@@ -0,0 +1,182 @@
+
+You also need `gh` CLI installed and authenticated:
+```bash
+brew install gh
+gh auth login
+```
+
+---
+
+## Project structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)